src/test/resolver/test_getaddrinfo.c

```diff
@@ -168,6 +168,7 @@ bool addrinfo_equals(const struct addrinfo* lhs, const struct addrinfo* rhs) {
                    getaddrinfo_rv_string(buf2_##__LINE__, rv##__LINE__),                           \
                    (rv##__LINE__ == EAI_SYSTEM) ? strerror(errno) : "N/A");                        \
             g_test_fail();                                                                         \
+            return;                                                                                \
```

Comment:
should this also be added to the `assert_addrinfo_equals` above?

Reply:
We need to return from this one because it indicates that the output pointer is invalid, so the rest of the test can't run. The same isn't necessarily true for `assert_addrinfo_equals`.

Added a comment.